project.clj

```diff
@@ -40,6 +40,7 @@
                  [puppetlabs/kitchensink]
                  [puppetlabs/trapperkeeper]
                  [puppetlabs/i18n]
+                 [puppetlabs/trapperkeeper-filesystem-watcher "1.1.0"]
```

Comment:
Should we release clj-parent first and bump it here instead of fs-watcher?

Reply:
Yeah that's probably better. In that case we need to merge https://github.com/puppetlabs/clj-parent/pull/79 first.